rails/features/admin_works_with_projects.feature

```diff
@@ -26,7 +26,7 @@ Feature: Admin can work with projects
     And I fill in "admin_project[landing_page_content]" with "<h1>New project FooBar!</h1>"
     And I press "Save"
     Then I should be on the projects index page
-    And I should see "1 project"
+    And I should see "2 projects"
```

Comment:
This changed because the default data for feature tests now creates a default project so the materials collections it creates have a project.  So now this test which is also creating a project will see 2 projects not just 1.

Reply:
Update from the future (2021-06-01):

I am only seeing 1 project again.  I am going to see what might have changed in Rails 6.1 to explain this.  :/